test/unit/math/mix/scal/fun/falling_factorial_test.cpp

```diff
@@ -1,204 +0,0 @@
-#include <stan/math/mix/scal.hpp>
```

Comment:
These tests should not be removed.  I think there was confusion in the issue about mixing different autodiff types and using mixed autodiff types.  The latter we support, the former we do not.  So `foo(fvar<var>, double)` is OK, even though it uses the mixed type `fvar<var>`.  In contrast, `foo(var, fvar<double>)` is not OK, because it mixes two different autodiff types (`var` and `fvar<double>`) even though it doesn't use a mixed autodiff type.